index.html

```diff
@@ -75,108 +81,123 @@ <h1 class="primary-title font-family-poppins">I'm Belden.<br> Glad to see you!</
+                <ul class="canopy">
+                   <li class="canopy-title ">facebook</li>
+                   <li class="can-items">
+                     <span>• </span>  Full Stack Dev <span>• </span>
+                   </li>
+                   <li class="can-items">2015</li>
+                </ul>
+    
+                <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+    
+                <ul class="column-links">
+                   <li class="column-link"><a href="#">html</a></li>
+                   <li class="column-link"><a href="#">ruby on rails</a></li>
+                   <li class="column-link"><a href="#">css</a></li>
+                   <li class="column-link"><a href="#">javascript</a></li>
+                </ul>
+    
+    
+                <a href="" class="column-button"> see project</a>
 
-            <a href="" class="column-button"> see project</a>
+            </div>
          </div>
 
-         <div class="col-1 column-item">
+         <div class="col-1 column-item row-reverse">
             <img src="images\Snapshoot (1).jpg" alt="">
-            <h2 class="col-title">Multi-post <br> stories</h2>
+            <div class="column-detail">
+                
+            <h2 class="col-title">Uber Navigation</h2>
 
             <ul class="canopy">
-               <li class="canopy-title ">canopy</li>
+               <li class="canopy-title ">Uber</li>
                <li class="can-items">
-                 <span>• </span>  backend-dev <span>• </span>
+                 <span>• </span>  Lead Developer <span>• </span>
                </li>
-               <li class="can-items">2015</li>
+               <li class="can-items">2018</li>
             </ul>
 
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
 
             <ul class="column-links">
                <li class="column-link"><a href="#">html</a></li>
+               <li class="column-link"><a href="#">ruby on rails</a></li>
                <li class="column-link"><a href="#">css</a></li>
                <li class="column-link"><a href="#">javascript</a></li>
             </ul>
 
 
             <a href="" class="column-button">see project</a>
+            </div>
          </div>
 
            
         </section>
 
         <section class="about-me font-family-poppins">
            
-            <div class="about-details">
+            <div  class="about-details">
                 <h2 class="about-title">About <br> Myself</h2>
                 <p>
                     Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
```

Comment:
- [ ] I think it would be better to fix the `width` of the `languages` items in mobile version in order to follow the figma template. What do yo think? 🤔 Right now, the `languages` item's `width` is halved from what is given in figma template. I know you can do that 💪
- [ ] Also please make sure to fix button and border after `languages` as mentioned above 👆

## **_Your design 👇_**
![image](https://user-images.githubusercontent.com/106119105/214626106-cec16075-6d1b-4fdd-9a26-825d326010fc.png)

## **_FIgma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/214626182-1369183b-e889-48c6-98b8-c54b141b6f48.png)

**_Please note that this is for your `ul/li` in about section. I'm adding it here because I'm unable to select those line. I know you can fix it 💪_**